cli/fix/langs.go

```diff
@@ -18,7 +18,7 @@ package main
 import (
 	"github.com/bazelbuild/bazel-gazelle/language"
 	"github.com/bazelbuild/bazel-gazelle/language/bazel/visibility"
-	"github.com/bazelbuild/bazel-gazelle/language/go"
+	golang "github.com/bazelbuild/bazel-gazelle/language/go"
```

Comment:
nit: move alias import to a separate import section below

Reply:
Reverted, this was from messing around with the dependency upgrading stuff